app/controllers/ips_controller.rb

```diff
@@ -14,6 +16,8 @@ def create
         notice: "#{@ip.address} added, it will be active starting tomorrow"
       )
     else
+      @ip.destroy if @ip.location.ips.count.zero?
```

Comment:
Not sure I follow this behaviour.
If @ip failed to save, we call .destroy on it?

Reply:
Right, should be on `location`